### PR TITLE
New OSG Connect projects LSUHSC_CanavierLab, RPI_Brown

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

File: projects/LSUHSC_CanavierLab.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name: LSUHSC_CanavierLab
+Description: "Nonlinear dynamics of single neurons and networks"
+Department: Cell Biology and Anatomy
+FieldOfScience: Neuroscience
+Organization: Louisianna State University Health Sciences Center
```
Comment (matyasselmeci) [inline projects/LSUHSC_CanavierLab.yaml]:
```suggestion
Organization: Louisiana State University Health Sciences Center
```

**Comment by matyasselmeci** [comment]
Looks like a valid new organization but there might be a typo... see my comment.

**Comment by brianhlin** [comment]
@lmichael107 could you accept Mat's suggestion above?

**Comment by lmichael107** [comment]
Thanks @matyasselmeci. Spelling is hard.

**Pull request merged**